index.html

```diff
@@ -2423,7 +2782,6 @@ <h4 class="modal-title">Help</h4>
 
                 $('#chat').offset({ top: $('nav').height() + 5 });
                 $('#chat-toggle-button').offset({ top: $('nav').height() + 7 });
-                $('#chat').height(window.innerHeight - $('nav').height() - 85);
```

Comment:
Was moved to other locations, and is now dynamic due to the option to hide 'Send'.